### ISSUE TITLE
dataset/big_dataset/train.txt not available in the repository

### ISSUE THREAD

**Issue by sonebu** [ebubekirbbr/phishing_url_detection#1]
Thanks a lot for the article and the repo. Are there any plans on making the training data for the "big dataset" available?

**Comment by ebubekirbbr** [comment]
Of course, I will share the big dataset. Our article is currently under review, when our article is published in the journal, I will update the repo by adding the big dataset.

**Comment by Aravindambalavanan** [comment]
Hey can upload the big dataset ?

**Comment by ebubekirbbr** [comment]
Hi, the project moved to https://github.com/ebubekirbbr/dephides/ you can
find all the data in this project.

Ebubekir Büber




Aravind ***@***.***>, 27 Şub 2024 Sal, 18:30 tarihinde şunu
yazdı:
